crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -813,6 +828,22 @@ pub(crate) enum SyncInfo {
     FullySynced,
 }
 
+/// Holds information computed from the room account data `m.tag` events.
+#[derive(Clone, Debug, Default)]
+pub struct RoomNotableTags {
+    /// Whether or not the room is marked as favorite.
+    pub is_favorite: bool,
+}
+
+impl RoomNotableTags {
+    /// Computes the provided tags to create a `RoomNotableTags` instance.
+    pub fn new(tags: Option<Tags>) -> Self {
+        RoomNotableTags {
+            is_favorite: tags.map(|tag| tag.contains_key(&TagName::Favorite)).unwrap_or(false),
```

Comment:
nit: `map(...).unwrap_or(x)` can be rewritten as `map_or(x, ...)`

Reply:
Here it could even be: `.is_some_and(...)`.